recipes/raylib/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class RaylibConan(ConanFile):
+    name = "raylib"
+    description = "raylib is a simple and easy-to-use library to enjoy videogames programming."
+    license = "zlib"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib.html